rails-controller-testing.gemspec

```diff
@@ -18,11 +18,11 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.2.2'
 
-  s.add_dependency "actionpack", "~> 5.x", ">= 5.0.1"
-  s.add_dependency "actionview", "~> 5.x", ">= 5.0.1"
-  s.add_dependency "activesupport", "~> 5.x"
+  s.add_dependency "actionpack", ">= 5.0.1", "< 6.1.x"
```

Comment:
Is there a reason we need `"< 6.1.x"`?

Reply:
@composerinteralia I think being a bit conservative is good, since we don't know the future.

This will make the gem compatible up to 6.1.0.rc* but not 6.1.0 final. By the point that 6.1.0.alpha starts, I think we will already have another patch ready to bump it to, say, `< 6.2.x`, depends on how Rails core team decide on what's the next version should be.